application/event_processor/tests/test_event_processor.py

```diff
@@ -96,7 +109,11 @@ def test_send_changes(mock_invoke_lambda_function):
         "system": "Profile Updater (test)",
         "message": "Test message 1531816592293|@./",
         "service_id": "49016",
-        "changes": {"ods_code": "f0000", "phone": "0118 999 88199 9119 725 3", "website": "https://www.google.pl"},
+        "changes": {
+            "ods_code": "f0000",
```

Comment:
ods_code should probably be defined as a constant somewhere too

Reply:
Removed `ods_code` as we aren't changing ods code with a change request.